java/src/main/java/com/powsybl/dataframe/network/adders/DanglingLineDataframeAdder.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.DanglingLineAdder;
+import com.powsybl.iidm.network.Network;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class DanglingLineDataframeAdder extends AbstractSimpleAdder {
+
+    private static final List<SeriesMetadata> METADATA = List.of(
+            SeriesMetadata.stringIndex("id"),
+            SeriesMetadata.strings("voltage_level_id"),
+            SeriesMetadata.strings("bus_id"),
+            SeriesMetadata.strings("connectable_bus_id"),
+            SeriesMetadata.ints("node"),
+            SeriesMetadata.strings("name"),
+            SeriesMetadata.doubles("p0"),
+            SeriesMetadata.doubles("q0"),
+            SeriesMetadata.doubles("r"),
+            SeriesMetadata.doubles("x"),
+            SeriesMetadata.doubles("g"),
+            SeriesMetadata.doubles("b")
+    );
+
+    @Override
+    public List<List<SeriesMetadata>> getMetadata() {
+        return Collections.singletonList(METADATA);
+    }
+
+    @Override
+    public void addElement(Network network, UpdatingDataframe dataframe, int indexElement) {
+        DanglingLineAdder adder = network.getVoltageLevel(dataframe.getStringValue("voltage_level_id", indexElement)
+                .orElseThrow(() -> new PowsyblException("voltage_level_id is missing"))).newDanglingLine();
+        NetworkElementCreationUtils.createInjection(adder, dataframe, indexElement);
```

Comment:
I think that we have to think about `g` and a `b` default values equals to zero.

Reply:
See discussion synthesis about default values in comment below.